### PR TITLE
Replace PaiEas LLM with LLI-integration and upgrade python to 3.11

### PR DESCRIPTION
**Pull request by wwxxzz**
1. Replace PaiEas LLM with LLI-integration
2. Replace MyFCDashScope LLM with LLI-integration-OpenAILike class
3. Upgrade python version to 3.11

### CODE REVIEW

**Review by moria97 — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -10,14 +10,14 @@ authors = []
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.10.0,<3.12"
```
Comment (moria97) [inline pyproject.toml:13]: 这里是不是可以改成<=3.12

File: pyproject.toml [hunk 2]
```diff
@@ -10,14 +10,14 @@ authors = []
...
+python = ">=3.11.0,<3.12"
 fastapi = "^0.110.1"
 uvicorn = "^0.29.0"
-llama-index-core = ">=0.10.29,<=0.10.39"
+llama-index-core = "0.10.57"
 llama-index-embeddings-openai = "^0.1.7"
 llama-index-embeddings-azure-openai = "^0.1.7"
 llama-index-embeddings-dashscope = "^0.1.3"
-llama-index-llms-openai = "^0.1.15"
+llama-index-llms-openai = "0.1.27"
```
Comment (moria97) [inline pyproject.toml:20]: 感觉这几个关联的都可以写成 "^xx.xx"形式，之前应该是担心和llama_index兼容性固定了某版本

File: pyproject.toml (hunk 1 above)
Comment (wwxxzz) [reply pyproject.toml:13]: 不太可行，protobuf有一个限制： For protobuf, a possible solution would be to set the `python` property to ">=3.7,<=3.12"

File: pyproject.toml (hunk 2 above)
Comment (wwxxzz) [reply pyproject.toml]: Fixed.

File: pyproject.toml (hunk 1 above)
Comment (moria97) [reply pyproject.toml:13]: 想起来了之前限制过protobuf的版本。。。

**Pull request merged**